entries/QUnit.module.xml

```diff
@@ -15,16 +15,31 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="grouped_tests" optional="true">
```

Comment:
Related to the comment below, "grouped" here is also odd. Maybe reuse the "nested" terminology?